templates/al2023/runtime/rootfs/etc/systemd/system/ebs-initialize-bin@.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=Initialize critical binary (%i) from an EBS filesystem
+After=basic.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ebs-initialize /usr/bin/%i
```

seems like a pretty niche script 😂 thoughts on just inlining it into the service?
```suggestion
ExecStart=/usr/bin/bash -c 'dd if="/usr/bin/%i" of=/dev/null bs=1M'
```

---

I wanted it to be testable/benchmarkable without the systemd dependency. The hardcoded “/usr/bin/“ part in the unit is because passing file paths as the instance name for template units is janky, and I didn’t want to move that down into the script